src/views/well-architected-framework/tutorial-view/index.tsx

```diff
@@ -53,14 +55,14 @@ export default function WellArchitectedFrameworkTutorialView({
 				key={slug}
 				{...(isInteractive && { labId: handsOnLab.id })}
 			>
-				<SidebarSidecarWithToc
-					headings={layoutProps.headings}
+				<SidebarSidecarLayout
+					sidecarSlot={<OutlineNavWithActive items={outlineItems.slice()} />}
 					breadcrumbLinks={layoutProps.breadcrumbLinks}
 					sidebarNavDataLevels={layoutProps.navLevels}
 					mainWidth="narrow"
 				>
 					<TutorialMeta
-						heading={{ slug: slug, text: name }}
```

Comment:
This fixes an issue where the first outline item did not link properly. Here we're passing the `tutorial` `slug` to `TutorialMeta`, which will not match the `slug` of the first heading. I've updated to build a simple `pageHeading` in `getStaticProps`, using the slug and text from the first `heading` element, and we now pass that through, so that the first outline item does link as expected.